src/screens.ts

```diff
@@ -20,8 +21,8 @@ module.exports = function Constructor(currentSettings) {
 				outputString += chalk.bold(`${settings.app.name}: `);
 			}
 
-			// Add version and a newline
-			outputString += `${settings.app.version}\n`;
+			// Add version (if we have one) and a newline
+			outputString += `${settings.app.version ? settings.app.version : ''}\n`;
```

Comment:
Slightly more shorthand way of doing this:

```suggestion
			outputString += `${settings.app.version || ''}\n`;
```